src/hooks/contract.js

```diff
@@ -59,7 +60,9 @@ const contractReducer = (currentState, action) => {
     case "CREATE_CORGI_SUCCESS":
       return {
         ...currentState,
+        loading: false,
```

Comment:
I use loading only for getting corgis, create function only have `creating`. please delete this line.

Reply:
Deleted it